Poker.BE/Poker.BE.Domain/Game/Room.cs

```diff
@@ -139,13 +139,13 @@ public void EndCurrentHand()
         /// UC027 Choose Play Move
         /// </summary>
         /// <see cref="https://docs.google.com/document/d/1OTee6BGDWK2usL53jdoeBOI-1Jh8wyNejbQ0ZroUhcA/edit#heading=h.8f3okxza6g2d"/>
-        public void ChoosePlayMove(Round.Move move)
+        public void ChoosePlayMove(Round.Move move, int amountToBet)
```

Comment:
so how do the user knows what are the options he can preform next?
or... he just tries something, and if its not possible we throw an exception?